fabric/gradle.properties

```diff
@@ -1,6 +1,6 @@
 minecraft_version=1.19.2
 yarn_mappings=1.19.2+build.28
-loader_version=0.14.12
+loader_version=0.14.19
```

Comment:
I take it back, this wasn't nice. I tested it locally and this seems to break the compat module, not necessarily just arrp, as this is loaded in the core module as well.

Reply:
reeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee